src/components/PledgedCollectiveCard.js

```diff
@@ -0,0 +1,223 @@
+import Currency from './Currency';
+import Link from './Link';
+import Logo from './Logo';
+import StyledLink from './StyledLink';
+import styled from 'styled-components';
+import StyledCard from './StyledCard';
+import { Box } from '@rebass/grid';
+import themeGet from '@styled-system/theme-get';
+
+const CardHeader = styled(Box)`
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  height: 12rem;
+`;
+
+const LogoBorder = styled.div`
+  height: 50%;
+  border-bottom: 1px solid;
+  border-color: ${themeGet('colors.black.200')};
+`;
+
+const LogoContainer = styled.div`
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+`;
+
+const CardBody = styled.div`
+  padding: 1rem;
+  min-height: 11rem;
+  height: 220px;
+`;
+
+const CardText = styled.div`
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const CollectiveNameText = styled(CardText)`
+  min-height: 20px;
+  font-size: 14px;
+  margin: 5px;
+  font-weight: 700;
+  text-align: center;
+  color: #303233;
+  white-space: nowrap;
+`;
+
+const CollectiveDescriptionText = styled(CardText)`
+  font-weight: bold;
+  text-align: center;
+  color: #787d80;
+  font-size: 1rem;
+  line-height: 1.3;
+  margin: 5px;
+  text-transform: uppercase;
+`;
+
+const CollectiveWebsiteText = styled(CardText)`
+  font-weight: normal;
+  text-align: center;
+  color: #0096f9;
+  font-size: 1.2rem;
+  line-height: 1.3;
+  margin: 0px;
+`;
+
+const MakePledgeButtonContainer = styled.div`
+  margin: 30px;
+  text-align: center;
+`;
+
+const ClaimCollectiveButtonContainer = styled.div`
+  margin: 30px;
+  text-align: center;
+`;
+
+const CardFooter = styled.div`
+  font-size: 1.1rem;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  border-top: 1px solid;
+  border-color: ${themeGet('colors.black.200')};
+`;
+
+const StatsContainer = styled.div`
+  display: flex;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  justify-content: space-around;
+`;
+
+const StatsItemContainer = styled.div`
+  height: 100%;
+  width: 50%;
+`;
+
+const StatsItemContainerWithBorder = styled(StatsItemContainer)`
+  border-left: 1px solid;
+  border-color: ${themeGet('colors.black.200')};
+`;
+
+const StatsItem = styled.div`
+  text-align: center;
+  margin: auto;
+`;
+
+const StatsItemValue = styled(StatsItem)`
+  font-weight: bold;
+  text-align: center;
+  color: #303233;
+  font-size: 1rem;
+  margin: 3px 2px 0px;
+`;
+
+const StatsItemLabel = styled(StatsItem)`
+  font-size: 1rem;
+  text-align: center;
+  font-weight: 300;
+  color: #303233;
+`;
+
+class PledgedCollectiveCard extends React.Component {
+  static propTypes = {
+    collective: PropTypes.object.isRequired,
+    membership: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { collective } = this.props;
+
+    const pledgeStats = collective.pledges.reduce(
```

Comment:
CollectiveInterface's `orders` field doesn't have `GraphQLNonNull` so let's be safe here

```suggestion
    const pledgeStats = (collective.pledges || []).reduce(
```